galaxy.yml

```diff
@@ -17,11 +17,12 @@ readme: README.md
 # @nicks:irc/im.site#channel'
 authors:
 - bthomass@redhat.com
+- jpisciot@redhat.com
 
 
 ### OPTIONAL but strongly recommended
 # A short summary description of the collection
-description: Event Driven Ansible
+description: Event-Driven Ansible
```

Comment:
Is the use of hyphen correct here? If so, we should update the rest of the doc to be consistent.

Reply:
yes it is. branding requirement